integration-tests/utils/HACBSApplications.ts

```diff
@@ -0,0 +1,96 @@
+import { applicationDetailPagePO } from '../support/pageObjects/createApplication-po';
+import { componentsTabPO } from '../support/pageObjects/hacbs-po';
+import { AddComponentPage } from '../support/pages/AddComponentPage';
+import { ComponentPage } from '../support/pages/ComponentsPage';
+import { AddIntegrationTestPage } from '../support/pages/hacbs/AddIntegrationTestPage';
+import { CreateBuildPage } from '../support/pages/hacbs/CreateBuildPage';
+import { Applications } from './Applications';
+import { Common } from './Common';
+
+export class HACBSApplications {
+    static deleteApplication(applicationName: string) {
+        Applications.deleteApplication(applicationName);
+    }
+
+    static createApplication(name: string) {
+        Applications.createApplication(name);
+    }
+
+    static createComponent(publicGitRepo: string, componentName: string) {
+        addComponentStep(publicGitRepo);
+        reviewComponentsStep(componentName);
+        createBuildStep();
+        addIntegrationTestStep();
+    }
+
+    static createdComponentExists(componentName: string, applicationName: string) {
+        this.goToComponentsTab();
+
+        Common.verifyPageTitle(applicationName);
+        Common.waitForLoad();
+        this.getComponentListItem(componentName).should('exist');
+    }
+
+    static getComponentListItem(application: string) {
+        return cy.contains(applicationDetailPagePO.item, application, { timeout: 60000 });
+    }
+
+    static goToComponentsTab() {
+        cy.get(componentsTabPO.clickTab).click({force: true});
```

Comment:
I'm not sure if the `force` is needed there, I tried locally and passed even without it.

Reply:
updated, thanks!